sdk/core/src/main/java/com/klaviyo/coresdk/lifecycle/KlaviyoLifecycleMonitor.kt

```diff
@@ -0,0 +1,44 @@
+package com.klaviyo.coresdk.lifecycle
+
+import android.app.Activity
+import android.app.Application
+import android.os.Bundle
+
+/**
+ * Service for monitoring the application lifecycle and network connectivity
+ */
+internal object KlaviyoLifecycleMonitor : LifecycleMonitor, Application.ActivityLifecycleCallbacks {
+
+    private var activitiesActive = 0
```

Comment:
nit: `activeActivities`

Reply:
Took me like 4 tries to even see the difference there. But yeah thats better.